### PR TITLE
Separate Alamofire/URLSession managers to different targets

### PR DESCRIPTION
**Pull request by magyarosibotond**
#3 Resolves

- [x] Create SPM target for Alamofire / URLSession managers.
- [x] Create subspec for the same targets.

### CODE REVIEW

**Comment by zierka** [comment]
Do we need to add the Alamofire source files to the Example project?

**Comment by magyarosibotond** [comment]
@zierka Alamofire was used since the URLSession driver is not implemented yet.

**Comment by zierka** [comment]
@magyarosibotond I mean can't we install it through cocoapods?

**Comment by magyarosibotond** [comment]
@zierka Alamofire is only added (to the lock file) since RestBird/Alamofire (the specific driver) depends on Core and Alamofire.

File: Example/Source/ViewController.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    }
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+        let cell = tableView.dequeueReusableCell(withIdentifier: "cell", for: indexPath)
+        cell.textLabel?.text = beers[indexPath.row].name
+        return cell
+    }
+}
+
+extension ViewController: UITableViewDelegate {
```
Comment (zierka) [inline Example/Source/ViewController.swift:53]: Empty stuff

File: Sources/RestBird-Alamofire/AlamofireSessionManager.swift [hunk 2]
```diff
@@ -31,22 +34,36 @@ class AlamofireSessionManager: SessionManager {
         }
     }
 
-    func performUploadTask<Request: UploadRequest>(request: Request, completion: @escaping (Result<Data>) -> Void) {
+    // MARK: - Private
+
+    private func urlWithBaseUrl<Request: RestBird.Request>(_ baseUrl: String, request: Request) -> String {
```
Comment (zierka) [inline Sources/RestBird-Alamofire/AlamofireSessionManager.swift:39]: This needs improvement in the future :)

**Review by zierka — APPROVED** [review]
It's ok for an initial implementation

File: Sources/RestBird-Alamofire/AlamofireSessionManager.swift (hunk 2 above)
Comment (magyarosibotond) [reply Sources/RestBird-Alamofire/AlamofireSessionManager.swift:39]: Noted

**Pull request merged**